src/main/java/com/flab/quicktogether/alarm/kakao/KaKaoAlarmSendService.java

```diff
@@ -0,0 +1,32 @@
+package com.flab.quicktogether.alarm.kakao;
+
+import com.flab.quicktogether.alarm.service.AlarmSendService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.RestTemplate;
+
+@Slf4j
+public class KaKaoAlarmSendService implements AlarmSendService<HttpEntity> {
+
+    @Autowired
+    private RestTemplate restTemplate;
```

Comment:
RestTemplate는 [Deprecated 예정](https://www.linkedin.com/pulse/resttemplate-deprecated-since-spring-5-andrzej-korcz)입니다.
가급적 WebClient나 Open Feign을 사용해보시면 좋을 것 같습니다~

Reply:
넵 알겠습니다.
장단점을 찾아보고 바꿔보도록 하겠습니다.